packages/core/src/2d/sprite/Sprite.ts

```diff
@@ -108,6 +111,19 @@ export class Sprite extends RefObject {
     }
   }
 
+  /**
+   * Only used in the atlas!!!
+   * Offset of Spirte in the atlas from the original picture.
+   */
+  get offset(): Vector2 {
+    return this._offset;
+  }
+
+  set offset(value: Vector2) {
```

Comment:
sprite have atlasRegion, do not need!
you use atlas only, but named offset, API name design like this is not good!

Reply:
@eyworldwide Hi Bob! In fact, we only have `SpriteAtlas`, not `AltasSprite`, `SpriteAtlas` is a collection of `Sprite`, similar to a tool asset, so inheritance should not be used.